### PR TITLE
[nng] Update to 1.3.2

### PR DESCRIPTION
**Pull request by myd7349**
**Describe the pull request**

- What does your PR fix?
Update `nng` to 1.3.2 (bug fixes).

- Which triplets are supported/not supported? Have you updated the CI baseline?
N/A. No.

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
Yes

### CODE REVIEW

File: ports/nng/portfile.cmake [hunk 1]
```diff
@@ -1,30 +1,28 @@
...
+    tools NNG_ENABLE_NNGCAT
 )
 
 vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}
-    PREFER_NINJA
-    OPTIONS ${FEATURE_OPTIONS}
-        -DCMAKE_DISABLE_FIND_PACKAGE_Git=TRUE
-        -DNNG_STATIC_LIB=${NNG_STATIC_LIB}
+    #PREFER_NINJA
```
Comment (NancyLi1013) [inline ports/nng/portfile.cmake:17]: Why do you disable `PREFER_NINJA` here?

Comment (myd7349) [reply ports/nng/portfile.cmake:17]: > 
> 
> Why do you disable `PREFER_NINJA` here?

It failed to install `nng` on my PC before when `ninja` is enabled. I will test it again.

**Review by NancyLi1013 — APPROVED** [review]

**Comment by NancyLi1013** [comment]
Looks work fine now. 
Have you tested these features in this PR?

**Comment by myd7349** [comment]
> 
> 
> Looks work fine now.
> Have you tested these features in this PR?

Yes.

File: ports/nng/portfile.cmake [hunk 2]
```diff
@@ -1,30 +1,28 @@
...
 
 vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}
     PREFER_NINJA
-    OPTIONS ${FEATURE_OPTIONS}
-        -DCMAKE_DISABLE_FIND_PACKAGE_Git=TRUE
-        -DNNG_STATIC_LIB=${NNG_STATIC_LIB}
+    OPTIONS
         -DNNG_TESTS=OFF
         -DNNG_ENABLE_NNGCAT=OFF
```
Comment (ras0219-msft) [inline ports/nng/portfile.cmake:20]:
```suggestion
```
This is provided by the `vcpkg_check_features()` above

**Review by ras0219-msft — CHANGES_REQUESTED** [review]
LGTM once my comment is addressed. Thanks for the PR!

**Comment by ras0219-msft** [comment]
Thanks for the PR!

**Pull request merged**